frontend/src/services/notebookService.ts

```diff
@@ -86,15 +86,15 @@ const assembleNotebook = (data: StartNotebookData): Notebook => {
   const resources: NotebookResources = { ...notebookSize.resources };
   const tolerations: NotebookToleration[] = [];
   let affinity: NotebookAffinity = {};
+  if (!resources.limits) {
```

Comment:
Took this logic out of the gpus > 0 condition to make sure the gpu is set to 0 if it previously was 1, otherwise the notebook resources just stay at whatever was there previously. Maybe the issue would look better solved higher up in the code but this was simplest at the time.